io.github.ja2_stracciatella.JA2-Stracciatella.yaml

```diff
@@ -0,0 +1,90 @@
+app-id: io.github.ja2_stracciatella.JA2-Stracciatella
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.rust-stable"
+command: ja2-launcher-wrapper
+rename-desktop-file: ja2-stracciatella.desktop
+rename-icon: ja2-stracciatella
+finish-args:
+  - '--share=network'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--share=ipc'
+  - '--device=dri'
+  - '--socket=pulseaudio'
+  # Persist ja2 setting files
+  - '--persist=.ja2'
+  # And allow some filesystem access to let JA2 read the game files
+  - '--filesystem=xdg-documents'
+modules:
+  - modules/string_theory.json
+  - modules/rapidjson.json
+  - modules/fltk.json
+  - modules/lua.json
+  - modules/sol2.json
```

Comment:
I'll reiterate: there is no reason to have these split out of the main manifest. Also it's better to use the lua in shared-modules.

Reply:
As said, JA2 doesn't build with lua from shared-modules.  And personally I prefer to have things split up for readability, so unless it's mandated by some kind of policy to have everything in one file, I'd like to keep it this way.

Reply:
it's not readable split like that.

Reply:
That's just your opinion 🤷🏿  I wrote this manifest this way, so you'll understand that my opinion is different.

I mean no offence, but this whole review feels pretty arbitrary… whether this manifest is split up or not is matter of taste and style, and as long as there's no official style guide mandated by some kind of policy for flathub I don't understand why the way the manifest is written is up for debate.

Reply:
Long story short, it will me miserable once you come to us with some technical problem. Sure, you wrote it that way and it looks fine to you, but we've reviewed over 1000 manifests and splitting modules only gives you an illusion of being more readable.

Reply:
Okay, sure, what do I know 🤷🏿  

I'll abandon this pull request; I'm sorry for having wasted your time, but this isn't fun anymore.  I just don't need to put of with this way of review and this kind of comments.  Not in my own free time when I could just spent time with my family instead.  Not when I can just have Github build this flatpak for me for my own use. 🤷🏿

I've unsubscribed from this pull request; please do feel free to close it.

Reply:
This does work both ways @lunaryorn. This is an open source project, maintained by volunteers. Writing an extensive documentation is a skill people who're contributing right now generally don't have. If we can't agree on making some changes now, what does it tell me about long-term maintenance and cooperation with someone? I assure you I also have more satisfying things to do in my free time than to bicker about YAML files.

Reply:
> If we can't agree on making some changes now, what does it tell me about long-term maintenance and cooperation with someone?

I don't know what this tells you 🤷🏿   I for my part don't see how the way this manifest is split up is related to my commitment about long-term maintenance of this manifest.  I could just as well follow all your whims and still bail out of maintenance right after it was accepted 🤷🏿 

As for cooperation, well, I don't feel like we're cooperating here, to be honest.  As I see it, you expect me to follow all your whims about the manifest and then expect _me_ to maintain long-term.  And it really just looked like whims to me:  There are plenty of manifests right here on flathub which are split into multiple module files (in fact I only learned about this feature by reading other manifests here on flathub); why is this okay for some manifests but not for this one?

That's all just not my idea of cooperation, and didn't sit well with me.  Being told that I suffer an "illusion of (it) being more readable" didn't help 🙂 That's just where it stops being fun 🤷🏿 

But see, we don't have to talk this out.  Your turf, your way, your rules 🤷🏿   But I don't have to play here in this way, so I'll just walk away, and maintain this manifest [elsewhere](https://github.com/lunaryorn/io.github.ja2_stracciatella.JA2-Stracciatella).  I don't mind 🙂